src/realm-roles/RolesList.tsx

```diff
@@ -34,11 +34,17 @@ type RolesListProps = {
   ) => Promise<RoleRepresentation[]>;
 };
 
-const RoleLink = ({ role }: { role: RoleRepresentation }) => {
+const RoleLink = ({
+  children,
+  role,
+}: {
+  role: RoleRepresentation;
+  children: ReactNode;
+}) => {
```

Comment:
You can use the `FunctionComponent` type here from React, which already includes the correct type for `children`.

```suggestion
type RoleLinkProps = {
  role: RoleRepresentation;
};

const RoleLink: FunctionComponent<RoleLinkProps> = ({ children, role }) => {
```